xgboost_ray/main.py

```diff
@@ -42,6 +43,7 @@
     RAY_INSTALLED = True
 except ImportError:
     ray = get_node_ip_address = Queue = Event = ActorHandle = logger = None
+    PublicAPI = DeveloperAPI = lambda fn: fn
```

Comment:
This will throw an exception with the keyword argument, as in `PublicAPI(stability="beta")`:
```
>>> f = lambda fn: fn
>>> @f(stability="beta")
... def a():
...     return
... 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: <lambda>() got an unexpected keyword argument 'stability'
```

Reply:
You can copy the dummy decorator in the sklearn.py

Reply:
Thanks!